odoo/models.py

```diff
@@ -1924,20 +1928,22 @@ def _read_group_select(self, aggregate_spec, query):
         if aggregate_spec == '__count':
             return 'COUNT(*)', []
 
-        fname, func = parse_read_group_spec(aggregate_spec)
+        fname, property_name, func = parse_read_group_spec(aggregate_spec)
+
+        fullname = f"{fname}.{property_name}" if property_name else fname
```

Comment:
sql_fname ? access_fname ? Don't like "fullname" as it hides a bit why we have two variables.